controllers/articles-controller.js

```diff
@@ -26,5 +26,15 @@ function getArticleById(req,res,next){
     })
 }
 
+function getArticles(req, res , next) {
+  selectArticles().then((articles) => {
+    res.status(200).send({ articles })
+  })
+  .catch((err) => {
+    console.log(err)
```

Comment:
don't forget to clean up your console logs once you've finished developing the endpoint 🧹